q2_moshpit/_utils.py

```diff
@@ -9,7 +9,7 @@
 from typing import List
 
 
-def run_command(cmd, env=None, verbose=True, pipe=False):
+def run_command(cmd, env=None, verbose=True, pipe=False, **kwargs):
```

Comment:
Uh oh, let's not do that - it introduces security concerns, as technically it would allow you to run anything in the users' system, _particularly_ if you use the _shell_ flag - see below.

Reply:
Sorry for the dirty workaround. I need to set the working directory for the command. This could be done simply by including `cd wd &&...` but it can also be done with the `cwd` parameter in `subproccess.run`. What do you think about adding this to `run_command` function? It might be useful in future use cases. 

I also added this `colorify` function which colors the print statement to the terminal. I think it's neat since it makes it easier to read the output. Let me know what you think. I got it from the eggnog scripts so I guess if we use them we should give them credits somewhere.